addon/templates/components/fixtable-grid.hbs

```diff
@@ -76,9 +76,13 @@
               {{#each columns as |column|}}
                 <td class={{column.cellClass}}>
                   {{#if column.component}}
-                    {{!-- render the passed-in component --}}
-                    {{component column.component dataRow=row columnDef=column}}
-                  {{else}}
+                        {{#if column.clickable}}
+                            {{component column.component dataRow=row column=column click=(action "clickCell" column.key row)}}
```

Comment:
It looks like you changed the `columnDef` property to `column` on this line, but left it as `columnDef` in the else block. Was that intentional? It seems like we should keep `columnDef` in both cases.